frontend/src/styles.css

```
@@ -4980,68 +5217,280 @@ input:focus-visible,
 
 /* ── Login page ─────────────────────────────────────────────────────────────── */
 
+@keyframes loginPanelIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "loginPanelIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)